test/steps/context.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package steps
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/cucumber/godog"
+)
+
+// registerContextSteps register all existing Context steps
+func registerContextSteps(s *godog.Suite, data *Data) {
+	s.Step(`^Check variable "([^"]*)" is an array of size (\d+)$`, data.checkVariableIsAnArrayOfSize)
```

Comment:
I don't see this step being used anywhere.

Reply:
I didn't want to remove them as we might need them in the future. 
But I guess it makes sense what you say, we have the git history anyway.